app/lib/features/chat/widgets/custom_input.dart

```diff
@@ -309,10 +310,13 @@ class _TextInputWidgetConsumerState extends ConsumerState<_TextInputWidget> {
           await chatRoomNotifier.typingNotice(false);
         }
       },
+      textInputAction: TextInputAction.send,
+      inputFormatters: [FilteringTextInputFormatter.deny(RegExp(r'\n'))],
```

Comment:
Pretty sure we _do not_ want to deny multi-line text ... you should still be able to do that via shift+enter ... Not sure this line is needed at all.
```suggestion
```